articles/azure/azs-how-add-disks.md

```diff
@@ -0,0 +1,247 @@
+
+# Set the VM Source Image
+$VirtualMachine =  Set-AzureRmVMSourceImage -VM $VirtualMachine -PublisherName $Image.PublisherName -Offer $Image.Offer -Skus $Image.Skus -Version 'latest'
+
+# Add Network Interface Card
+$VirtualMachine = Add-AzureRmVMNetworkInterface -Id $NetworkInterface.Id -VM $VirtualMachine
+
+# Set the OS Disk properties
+$OSDiskName = "OsDisk"
+$OSDiskUri = '{0}vhds/{1}-{2}.vhd' -f $StorageAccount.PrimaryEndpoints.Blob.ToString(), $VMName.ToLower(), $OSDiskName
+
+# Applies the OS disk properties
+$VirtualMachine = Set-AzureRmVMOSDisk -VM $VirtualMachine -Name $OSDiskName -VhdUri $OSDiskUri -CreateOption FromImage
+
+# Set additional disks properties
+$DataDiskVhd1 = "DataDisk1"
+$DataDisk1 = '{0}vhds/{1}-{2}.vhd' -f $StorageAccount.PrimaryEndpoints.Blob.ToString(), $VMName.ToLower(), $DataDiskVhd1
+
+$DataDiskVhd2 = "DataDisk2"
+$DataDisk2 = '{0}vhds/{1}-{2}.vhd' -f $StorageAccount.PrimaryEndpoints.Blob.ToString(), $VMName.ToLower(), $DataDiskVhd2
+
+$DataDiskVhd3 = "DataDisk3"
+$DataDisk3 = '{0}vhds/{1}-{2}.vhd' -f $StorageAccount.PrimaryEndpoints.Blob.ToString(), $VMName.ToLower(), $DataDiskVhd3
+
+# Applies the additional disks properties
+$VirtualMachine = Add-AzureRmVMDataDisk -VM $VirtualMachine -Name $DataDiskVhd1 -Caching 'ReadOnly' -DiskSizeInGB 10 -Lun 0  -VhdUri $DataDisk1 -CreateOption Empty
+$VirtualMachine = Add-AzureRmVMDataDisk -VM $VirtualMachine -Name $DataDiskVhd2 -Caching 'ReadOnly' -DiskSizeInGB 10 -Lun 1 -VhdUri $DataDisk2 -CreateOption Empty
+$VirtualMachine = Add-AzureRmVMDataDisk -VM $VirtualMachine -Name $DataDiskVhd3 -Caching 'ReadOnly' -DiskSizeInGB 10 -Lun 2 -VhdUri $DataDisk3 -CreateOption Empty
+
+# Create the virtual machine.
+Write-Host "Creating virtual machine"
+$NewVM = New-AzureRmVM -ResourceGroupName $RGName -Location $Location -VM $VirtualMachine
+$NewVM
+Write-Host "Virtual machine created successfully"
+```
+
+## Using PowerShell to add disks to an existing VM
+
+The following code will add new disks to VM's that already exists. It retrieves the VM from the resource group, modifies it and then updates it on the stack
+
+```powershell
+# Retrieves the VM
+$VirtualMachine = Get-AzureRmVM -ResourceGroupName "myResourceGroup" -Name "myVM"
+
+# Set additional disks properties
+$DataDiskVhd1 = "DataDisk4"
+$DataDisk1 = '{0}vhds/{1}-{2}.vhd' -f $StorageAccount.PrimaryEndpoints.Blob.ToString(), $VMName.ToLower(), $DataDiskVhd1
+
+# Applies the additional disks properties
+$VirtualMachine = Add-AzureRmVMDataDisk -VM $VirtualMachine -Name $DataDiskVhd1 -Caching 'ReadOnly' -DiskSizeInGB 10 -Lun 4  -VhdUri $DataDisk1 -CreateOption Empty
+
+# Updates the VM
+Update-AzureRmVM -ResourceGroupName "myResourceGroup" -VM $VirtualMachine
+```
+
+> [!IMPORTANT]
+> Remember each time you add a disk to increase the **LUN** (Logical Unit Number).
+
+## Next Steps
+
+Now you know how to add disks to VM's this guide will be needed in the following guides:
```

Comment:
Remove apostrophe

Reply:
Suggest changing this to something like:

This article has shown you how to add disks to VM, which you'll find useful when setting up disaster recovery for your VMs. For more information, see:

Reply:
done